### ISSUE TITLE
login broken

### ISSUE THREAD

**Issue by interbiznw** [SN4T14/capturebate-node#16]
Looks like chaturbate added captcha, and breaks some instances

**Comment by momo211** [comment]
captcha https://i.imgur.com/DIPEvFC.png prevents script from logging in

Is there a way to fix it ?

Or maybe, at least, a way handle models with a .txt list (so no need to log in)

Regards,

**Comment by ramtico** [comment]
Same problem here.  It seems it doesn't work anymore.

**Comment by sstativa** [comment]
I don't think that captcha is "permanent". I suppose it appears only if the user violates some limits, for example, if the user logins to often.

However, I might be wrong. Try to log in on different devices and using different IPs. If the captcha persists then this is not a good sign.
